docs/product-and-design/copy-style.md

```diff
@@ -0,0 +1,46 @@
+# Copy terms and style
+
+Use the following terms and style rules for Benefits application English and Spanish copy. Refer to the [Cal-ITP Editorial Style Guide](https://docs.google.com/document/d/1yG2jc_aMPSGc2wqvazKGgFIeV-aDDDsJIuONhUg0MOI/edit) and [Language Matters: Cal-ITP Terminology](https://docs.google.com/document/d/1ud73Qaa7k6I5l8MwlQrZd8iOo7_3shZHy_gCxfAe0eY/edit#heading=h.keo0rpgsdx29) for further guidance.
```

Comment:
Thanks for linking to these. Reading through them again and more closely, I realized we've had copy questions in the past that could've been answered by these, so I think it'll be useful to have more awareness of these guidelines.